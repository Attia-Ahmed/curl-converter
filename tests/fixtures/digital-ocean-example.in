curl -H "Authorization: Bearer abcdef0123456789" -H "Content-Type: application/json" -d '{"name": "example.com", "ip_address": "127.0.0.1"}' -X POST "https://api.digitalocean.com/v2/domains"
