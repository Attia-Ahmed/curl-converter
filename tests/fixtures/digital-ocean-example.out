Http::withBody('{"name": "example.com", "ip_address": "127.0.0.1"}')
    ->withToken('abcdef0123456789')
    ->post('https://api.digitalocean.com/v2/domains')
